src/components/Repos/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import RepoCard from '../RepoCard';
+
+function Repos() {
```

Comment:
duplicated code? This Repo component is called in Home component but in the home component, it contains the exact same code as well as the Repo Component.

Although the logic is spot on!

Reply:
Yes, that's tight. Thank you for spotting it. I cleaned up the Home page and merged changes to main.